vochain/scrutinizer/vote.go

```diff
@@ -59,7 +59,18 @@ func (s *Scrutinizer) GetEnvelope(nullifier []byte) (*models.VoteEnvelope, []byt
 	if envelope == nil {
 		return nil, nil, fmt.Errorf("transaction is not an Envelope")
 	}
-	return envelope, stx.Signature, nil
+	envelope.Nullifier = voteRef.Nullifier
+	return &types.EnvelopePackage{
+		Nonce:                envelope.Nonce,
+		ProcessId:            envelope.ProcessId,
+		VotePackage:          envelope.VotePackage,
+		Nullifier:            nullifier,
```

Comment:
Do we need nullifier? It's the parameter used to fetch the envelope so the caller already have it.